blocks/author-list/author-list.css

```diff
@@ -71,9 +75,9 @@ div.author-list-container {
 }
 
 @media (min-width: 1200px) {
-  .section.experts-page-heading>div,
-  .section.experts-page-editorial-staff>div,
-  .section.author-list-container>div {
+  .section.experts-page-heading > div,
+  .section.experts-page-editorial-staff > div,
+  .section.author-list-container > div {
```

Comment:
@synox Did you fix this using tool?  my stylint did not do this for me :)

Reply:
This one I did manually.